dev/02_dev.R

```diff
@@ -0,0 +1,103 @@
+# Building a Prod-Ready, Robust Shiny Application.
+# 
+# README: each step of the dev files is optional, and you don't have to 
+# fill every dev scripts before getting started. 
+# 01_start.R should be filled at start. 
+# 02_dev.R should be used to keep track of your development during the project.
+# 03_deploy.R should be used once you need to deploy your app.
+# 
+# 
+###################################
+#### CURRENT FILE: DEV SCRIPT #####
+###################################
+
+# Engineering
+
+## Dependencies ----
+## Add one line by package you want to add as dependency
+usethis::use_package( "thinkr" )
+usethis::use_package( "rpecanapi" )
+usethis::use_package( "shinydashboard" )
+usethis::use_package( "shinydashboardPlus" )
+usethis::use_package( "tidyr" )
+usethis::use_package( "tibble" )
+usethis::use_package( "purrr" )
+usethis::use_package( "httr" )
+usethis::use_package( "glue" )
+
+## Add modules ----
+## Create a module infrastructure in R/
+golem::add_module( name = "dashboard" ) # Name of the module
+golem::add_module( name = "test" ) # Name of the module
+golem::add_module( name = "report" )
+## Add helper functions ----
+## Creates fct_* and utils_*
+golem::add_fct( "helpers" ) 
+golem::add_utils( "helpers" )
+
+## External resources
+## Creates .js and .css files at inst/app/www
+golem::add_js_file( "script" )
+golem::add_js_handler( "handlers" )
+golem::add_css_file( "custom" )
+
+## Add internal datasets ----
+## If you have data in your package
+usethis::use_data_raw( name = "my_dataset", open = FALSE ) 
+
+## Tests ----
+## Add one line by test you want to create
+usethis::use_test( "app" )
+
+# Documentation
+
+## Vignette ----
+usethis::use_vignette("statusboard")
+devtools::build_vignettes()
+
+## Code Coverage----
+## Set the code coverage service ("codecov" or "coveralls")
+usethis::use_coverage()
+
+# Create a summary readme for the testthat subdirectory
+covrpage::covrpage()
+
+## CI ----
+## Use this part of the script if you need to set up a CI
+## service for your application
+## 
+## (You'll need GitHub there)
+usethis::use_github()
+
+# GitHub Actions
+usethis::use_github_action() 
+# Chose one of the three
+# See https://usethis.r-lib.org/reference/use_github_action.html
+usethis::use_github_action_check_release() 
+usethis::use_github_action_check_standard() 
+usethis::use_github_action_check_full() 
+# Add action for PR
+usethis::use_github_action_pr_commands()
+
+# Travis CI
+usethis::use_travis() 
+usethis::use_travis_badge() 
+
+# AppVeyor 
+usethis::use_appveyor() 
+usethis::use_appveyor_badge()
+
+# Circle CI
+usethis::use_circleci()
+usethis::use_circleci_badge()
+
+# Jenkins
+usethis::use_jenkins()
+
+# GitLab CI
+usethis::use_gitlab_ci()
```

Comment:
Not clear to me why the app would depend on Travis and Jenkins, which we haven't used in years. Also, what are AppVeyor, Circle CI, and GitLab CI and what are they being used for.